mlir/test/Target/LLVMIR/Import/function-attributes.ll

```diff
@@ -344,6 +344,72 @@ declare void @func_attr_no_signed_zeros_fp_math_true() "no-signed-zeros-fp-math"
 ; CHECK-SAME: attributes {no_signed_zeros_fp_math = false}
 declare void @func_attr_no_signed_zeros_fp_math_false() "no-signed-zeros-fp-math"="false"
 
+; // -----
+
+; CHECK-LABEL: @func_attr_denormal_fp_math_ieee
+; CHECK-SAME: attributes {denormal_fp_math = "ieee"}
+declare void @func_attr_denormal_fp_math_ieee() "denormal-fp-math"="ieee"
```

Comment:
Since these are string attributes there is probably not much value in testing different string values for every attribute (i.e. ieee, preserve-sign)? Or is there specific reason for testing the different strings? Instead I would have a test for every attribute with possible different strings? So three tests here and three tests in the file below which exercises the lowering to LLVM IR.

Reply:
> Or is there specific reason for testing the different strings?

These different strings are the general cmd options in codegen such as [denormalFpMath related](https://github.com/llvm/llvm-project/blob/main/llvm/lib/CodeGen/CommandFlags.cpp#L259) and [fp-contract](https://github.com/llvm/llvm-project/blob/main/llvm/lib/CodeGen/CommandFlags.cpp#L301). So i tested all these general cmd options for functional completeness.

Reply:
Ok! I saw there is already prior art above. 

I think the point I tried to make is that testing different values would make a lot of sense if the stored value would be an enum. Then we want to test that all enum values exist and are handled properly. In this case, we store a string so if one works all other strings should work as well (there are no separate code paths for different enum values). For maintenance reasons, I would thus keep the test set minimal if there is no extra functionality (in the sense of code coverage) that is tested.